http_service/bugbug_http/app.py

```diff
@@ -275,6 +299,23 @@ def get_bugs_last_change_time(bug_ids):
     return bugs
 
 
+def get_github_issues_update_time(
+    owner: str, repo: str, issue_nums: Sequence[int]
+) -> dict:
+    header = {"Authorization": "token {}".format(GITHUB_TOKEN)}
+    repo_url = "https://api.github.com/repos/{}/{}/issues/".format(owner, repo)
```

Comment:
Nit: use a f-string such as `f"https://api.github.com/repos/{owner}/{repo}/issues/"` instead of .format()